R/main.R

```diff
@@ -3,18 +3,20 @@ main_api_args <- function(args = commandArgs(TRUE)) {
   hintr_api [options] [<queue_id>]
 
 Options:
---workers=N   Number of workers to spawn [default: 2]
---port=PORT   Port to use [default: 8888]"
+--workers=N              Number of workers to spawn [default: 2]
+--port=PORT              Port to use [default: 8888]
```

Comment:
please use `--results-dir=PATH` as the option (not the hyphen) and ensure to line up the spaces

Reply:
Fixed that too